File: Year_3/Semester_5/Mobile_Applications/flutter_app/lib/model/daily_report.dart
class DailyReport {
  String id;
  String title;
  String gratitude;
  String accomplishments;
  String shortcomings;
  String improvementAreas;
  int rating;
  DateTime date;

  DailyReport({
    required this.id,
    required this.title,
    required this.gratitude,
    required this.accomplishments,
    required this.shortcomings,
    required this.improvementAreas,
    required this.rating,
    required this.date,
  });
}

File: Year_3/Semester_5/Mobile_Applications/AndroidDB/app/src/main/java/com/example/androiddb/domain/model/DailyReport.kt
package com.example.androiddb.domain.model

import java.util.Date
import java.util.UUID

data class DailyReport(
    val id: UUID,
    val title: String,
    val gratitude: String,
    val accomplishments: String,
    val shortcomings: String,
    val improvementAreas: String,
    val rating: Int,
    val date: Date
)